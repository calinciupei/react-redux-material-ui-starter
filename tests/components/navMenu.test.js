import React from 'react';
import { shallow } from 'enzyme';

import NavMenu from '../../src/components/navMenu/navMenu.component';

let context;

const createComponent = () => shallow(<NavMenu />);

describe('<NavMenu />', () => {
  context = createComponent();

  it('should render correctly', () => {
    expect(context.dive()).toMatchSnapshot();
  });
});
