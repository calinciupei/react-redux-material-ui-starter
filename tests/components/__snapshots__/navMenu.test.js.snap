// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NavMenu /> should render correctly 1`] = `
<div
  className="navigation"
>
  <div
    className="nav-menu"
  >
    <WithStyles(List)
      className="NavMenu-list-1"
    >
      <WithStyles(ListItem)
        className="NavMenu-listItem-2"
      >
        <WithStyles(Button)
          href="/"
        >
          Menu 1
        </WithStyles(Button)>
      </WithStyles(ListItem)>
      <WithStyles(ListItem)
        className="NavMenu-listItem-2"
      >
        <WithStyles(Button)
          href="about"
        >
          Menu 2
        </WithStyles(Button)>
      </WithStyles(ListItem)>
      <WithStyles(ListItem)
        className="NavMenu-listItemButton-3"
      >
        <WithStyles(Button)
          className="flatPrimary"
          color="primary"
          variant="outlined"
        >
          Sign In
        </WithStyles(Button)>
      </WithStyles(ListItem)>
      <WithStyles(ListItem)
        className="NavMenu-listItemButton-3"
      >
        <WithStyles(Button)
          color="primary"
          variant="contained"
        >
          Sign Up
        </WithStyles(Button)>
      </WithStyles(ListItem)>
    </WithStyles(List)>
  </div>
</div>
`;
